src/ext/data.js

```diff
@@ -64,9 +64,11 @@ export default function getData(env) {
         return dimension;
       },
       remove(dimension, data, hcHandler, idx) {
-        const { qColumnOrder, columnWidths } = hcHandler.hcProperties;
-        indexRemoved(qColumnOrder, idx);
-        columnWidths.splice(qColumnOrder[idx], 1);
+        if (!dimension.isAlternative) {
+          const { qColumnOrder, columnWidths } = hcHandler.hcProperties;
+          indexRemoved(qColumnOrder, idx);
+          columnWidths.splice(qColumnOrder[idx], 1);
```

Comment:
Is the removed dimension not represented in `qColumnOrder` and `columnWidths`?

Reply:
No it is not if it is alternative dimension it is not showed in the table.